Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Quote.hs

```diff
@@ -86,8 +87,8 @@ getQuotes searchRequestId = do
   paymentMethods <- getPaymentMethods searchRequest
   return $
     GetQuotesRes
-      { fromLocation = Location.makeSearchReqLocationAPIEntity searchRequest.fromLocation,
-        toLocation = Location.makeSearchReqLocationAPIEntity <$> searchRequest.toLocation,
+      { fromLocation = Location.makeLocationAPIEntity searchRequest.fromLocation,
+        toLocation = Location.makeLocationAPIEntity <$> lastMaybe searchRequest.toLocation,
```

Comment:
why this exists , we can put in location only why do we have searchLocation, bookingLocation separately when it is not required now